ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -242,20 +242,16 @@ export const allModules: AccessModule[] = [
   duccModule,
 ];
 
-// this function does double duty:
-// - returns appropriate text for completed and bypassed modules and null for incomplete modules
-// - because of this, truthy return values indicate that a module is either complete or bypassed
+const isCompleted = (status: AccessModuleStatus) => status && !!status.completionEpochMillis
+const isBypassed = (status: AccessModuleStatus) => status && !!status.bypassEpochMillis
+const isDone = (status: AccessModuleStatus) => isCompleted(status) || isBypassed(status)
```

Comment:
elsewhere (may be just the API) we have used "compliant" to indicate (completed or bypassed)

Reply:
open to naming ideas.  Can just leave it as "done" for now.